cl/simple_pages/templates/help/advanced_search.html

```diff
@@ -51,17 +57,27 @@ <h3>Grouped Queries and subqueries: <code>( )</code></h3>
 
-  <p>The question mark character (<strong>?</strong>) can be used similarly as a single letter wildcard. For example, this would find cases containing the word "immigrant" or "emmigration": <code>?mmigra*</code>
-  </p>
+  <p><code>*</code> can also be used inside words, where it acts as a single-character wildcard. For example, a query like <code>gr*mm*r</code> would match cases containing both "gr<strong>a</strong>mm<strong>a</strong>r" and "gr<strong>i</strong>mm<strong>e</strong>r".</p>
+  <p>The question mark character (<code>?</code>) can be used similarly as a single-character wildcard. Unlike <code>*</code>, it is allowed at the beginning of words. For example, this would find cases containing the word "immigrant" or "emmigration": <code>?mmigra*</code>.</p>
 
-  <p>Fuzzy search can be applied using the tilde character (<strong>~</strong>) after a word. This is an advanced parameter that allows searches for misspellings or different variations of a word's spelling. For example, searching for <code>immigrant~</code> would find words similar to "immigrant". Values can also be added after the tilde to indicate how similar different spellings must be. The default value, if none is given, is 0.5. Values can range between 0 and 1, with 1 being exact, and 0 being very sloppy. Fuzzy searches tend to broaden the result set, thus lowering precision, but also casting a wider net.
-  </p>
+  <p>Fuzzy search can be applied using the tilde character (<strong>~</strong>) after a word. This is an advanced parameter that allows searches for misspellings or variations in a word's spelling. For example, searching for <code>immigrant~</code> would find words similar to "immigrant." Values can also be added after the tilde to specify the maximum number of changes allowed, where a change refers to the insertion, deletion, substitution of a single character, or transposition of two adjacent characters. The default value, if none is given, is 2. Allowed values are 1 and 2.  Fuzzy searches tend to broaden the result set, thus lowering precision, but also casting a wider net.</p>
+
+  <h3 id="disallowed-expensive-wildcards">Disallowed Expensive Wildcards</h3>
+  <p>The following types of wildcard queries are disabled due to performance issues:</p>
+
+  <strong><code>*</code> at the beginning of terms</strong>
+  <p>Queries like <code>*ing</code> are disallowed because they require examining all terms in the index, which is highly resource-intensive.</p>
+
+  <strong>Multiple endings with <code>*</code> or <code>!</code> in short terms</strong>
+
+  <p>Queries that match multiple endings are only allowed if the base word has at least three characters. Therefore, queries like <code>a*</code>, <code>bc*</code>, <code>!a</code>, or <code>!bc</code> are disallowed due to performance issues.</p>
+  <p>Performing a query like these will throw an error with the message:</p>
+  <p><code>The query contains a disallowed expensive wildcard pattern.</code></p>
```

Comment:
```suggestion
  <p><code>The query contains a disallowed wildcard pattern.</code></p>
```

Reply:
Removed. Also from the section title, now just `Disallowed Wildcards`